Assets/Scripts/Player/PlayerController.cs

```diff
@@ -400,6 +400,11 @@ public void ForceCancelDash()
             TimeDashEnded = Time.time;
         }
 
+        /// <summary>
+        /// Force the player's velocity to 0.
+        /// </summary>
+        public void ForceCancelVelocity() => _velocity = Vector2.zero;
```

Comment:
could just give the player an impulse SimpleVelocityEffector that does this (or more accurately, does like `new Vector2(-0.0001, 1)` so the sign of the velocity is right for you force setting its direction)

could avoid both making the direction field public and having a velocity setter

Reply:
sure.